slackroll

```diff
@@ -14,10 +14,47 @@
 # this software. If not, see
 # <http://creativecommons.org/publicdomain/zero/1.0/>.
 #
-import anydbm
+import sys
+
+# BCS: Python 3 'dbm' == Python 2 'anydbm', both differ from Python 2's 'dbm'
+if sys.version_info[0] <= 2:
```

Comment:
Can we create a constant for this (' sys.version_info[0] <= 2' ) so we only have to define it once?

Reply:
Addressed in b31999b.